src/ManagerStandalone.sol

```diff
@@ -46,6 +46,18 @@ contract ManagerStandalone is IManagerStandalone, Manager, Implementation {
         IEndpointStandalone(endpoint).upgrade(newImplementation);
     }
 
+    /// @dev Transfer ownership of the Manager contract and all Endpoint contracts to a new owner.
+    function transferOwnership(address newOwner) public override onlyOwner {
+        super.transferOwnership(newOwner);
+        // loop through all the registered endpoints and set the new owner of each endpoint to the newOwner
+        address[] storage _registeredEndpoints = _getRegisteredEndpointsStorage();
+        _checkRegisteredEndpointsInvariants();
+
+        for (uint256 i = 0; i < _registeredEndpoints.length; i++) {
+            IEndpointStandalone(_registeredEndpoints[i]).initializeEndpoint();
```

Comment:
Why do we need to re-initialize here? We might run into some unintended behavior.
Why not expose a method that's `onlyManager` in the `EndpointStandalone` that allows transferring ownership directly?

Reply:
Yeah agreed, can we override `transferOwnership` in the endpoints to only allow calls from the manager?